src/openms/include/OpenMS/QC/Ms2IdentificationRate.h

```diff
@@ -69,6 +71,17 @@ namespace OpenMS
     /// container that stores results
     std::vector<IdentificationRateData> rate_result_;
 
+    /// returns number of all ms2 spectra in an MSExperiment
+    Size getMS2Count_(const MSExperiment& exp);
+
+    /// increases a 'counter' if PeptideIdentification is considered 'target' (target/decoy annotation needed)
```

Comment:
when referring to parameters, just `@p ...` to get proper highlighting, e.g. `@p ....`
but here, you can go classical
```
@param id ...
@param counter ...
``` 
to make it a bit more structured